.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "endOfLine": "lf",
+  "useTabs": false,
```

Comment:
Please: `"useTabs": true` because I like the way you can control the width of the tabs (tab distance) in editors

Reply:
GOT IT

Reply:
I changed it back. A little tip BTW, `useTabs:false` actually won't loose control of editor `Spaces:2`, it's only affect which indent type are using (Tabs or Spaces), but not how many spaces. Prettier will accept the `tabWidth` to do  the actual indention.

Reply:
You're right, because prettier controls the tabWidth it "shouldn't" matter, except if someone forks the project and don't run prettier, with tabs they get their 4 space tabs (or how many they have set up in their IDE), but with spaces it depends on the collaborator(s) that write the original code.